server/server.js

```diff
@@ -2,35 +2,62 @@ const express = require('express')
 const app = express()
 const port = 3000
 
-const bodyParser = require('body-parser');
-app.use(bodyParser.urlencoded({ extended: true }));
-
-const organizations = {}
-let currId = 0
-
-
+const organizations = {
+	0: { 'Name': 'Nandu Business', 'Net Worth': '6 Billion', 'Business Type': 'Real Estate Dealer' },
+	1: { 'Name': 'Sailesh Business', 'Net Worth': '20 Billion', 'Business Type': 'Food Dealer' }
+}
+const event = {
+	0: {'event': 'Teej', 'Location': 'niggerLand', 'Date': '9/11', 'Price': '$69' },
+	1: {'event': 'Christmas', 'Location': 'CuckLand',  'Date': '4/20', 'Price': '$69'}
+}
 
 app.get('/', (req, res) => res.send('Hello United Nepali!'))
 
 app.get('/health-check', (req, res) => res.send('OK'))
 
-app.listen(port, () => console.log(`Example app listening on port ${port}!`))
+app.get('/nepali-check', (req, res) => res.send('Bhaat Khaiyo?'))
 
 app.get('/organizations', (req, res) => res.send(organizations))
 
-app.post('/organizations', function(req, res){
-	let id_val = currId;
-	currId = currId + 1;
+app.get('/organizations_id', function (req, res) {
+	var id = req.query.id;
+	let org = organizations[id]
+	res.send(org)
+})
 
-	organizations[id_val] = {'Name':req.body.name};
-	res.sendStatus(200);
+app.delete('/organization_id', function (req, res) {
+    var id = req.query.id;
+    let org = organizations[id]
+    delete organizations[id]
+    res.sendStatus(200)

```

Comment:
Issue: https://github.com/nepalcodes/unitednepali/issues/11
Here's what you'd need to change to make it compliant to the issue:
```suggestion
    if (org) {
      res.status(200)
      res.send({
        "success": 1,
        "timestamp": new Date().getTime()
      })
    } else {
      res.status(500)
      res.send({
        "success": 0,
        "timestamp": new Date().getTime()
        "error": "Organization with id " + id + " does not exist!"
      })
   }
```
I just wrote this in a comment and haven't tested this out, so there might be some bugs here :P